source/index.html

```diff
@@ -1,12 +1,93 @@
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="main-container__wrapper wrapper">
+      <h1 class="wrapper__title">Функциональное питание для котов</h1>
+      <p class="wrapper__text">Занялся собой? Займись котом!</p>
+      <img src="img/images/general-img.jpg" width="280" height="270" alt="Корм для котов">
+      <button class="wrapper__button" type="button"><span>Подобрать программу</span></button>
+    </section>
 
-  </body>
+    <div class="main-container__advantages advantages">
+      <article class="advantages__slimming">
+        <img src="img/images/cat-index-1.svg" alt="кот">
+        <h2 class="advatages__title">Похудение</h2>
+        <p class="advantages__text">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim
+          поможет вашему питомцу сбросить лишний вес.</p>
+      </article>
+      <article class="advantages__weight-gain">
+        <img src="img/images/cat-index-2.svg" alt="кот">
+        <h2 class="advatages__title">Набор массы</h2>
+        <p class="advantages__text">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить
+          необходимые мышцы!</p>
+      </article>
+      </div>
+
+      <article class="main-container__description description">
+        <h2 class="description__title">Как это работает</h2>
+        <ul class="description__list">
+          <li class="description__item">
+            <img src="img/images/img-green-1.svg" width="60" height="60" alt="">
+            Функциональное питание
+            содержит только полезные
+            питательные вещества.</li>
+          <li class="description__item">
+            <img src="img/images/img-green-2.svg" width="60" height="60" alt="">
+            Выпускается в виде порошка,
+            который нужно лишь залить
+            кипятком и готово.</li>
+          <li class="description__item">
+            <img src="img/images/img-green-3.svg" width="60" height="60" alt="">
+            Замените один-два приема
+            обычной еды на наше
+            функциональное питание.</li>
+          <li class="description__item">
+            <img src="img/images/img-green-4.svg" width="60" height="60" alt="">
+            Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</li>
+        </ul>
+      </article>
+      <article class="main-container__examples examples">
+        <h2 class="examples__title">Живой пример</h2>
+        <p class="examples__text">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+      <table class="examples__description">
+        <dt>5кг</dt>
+        <dd>снижение веса</dd>
+        <dt>60 дней</dt>
+        <dd>затрачено времени</dd>
+      </table>
```

Comment:
Перемешалась таблица и dl/dt.

```suggestion
      <div class="examples__description">
      <dl>
        <dt>5кг</dt>
        <dd>снижение веса</dd>
      </dl>
      <dl>
        <dt>60 дней</dt>
        <dd>затрачено времени</dd>
      </dl>
      </div>
```